src/bin.rs

```diff
@@ -5,10 +5,33 @@ use std::path::Path;
 
 use lscolors::{LsColors, Style};
 
+#[cfg(all(
+    not(feature = "nu-ansi-term"),
+    not(feature = "gnu_legacy"),
+    not(feature = "ansi_term"),
+    not(feature = "crossterm")
+))]
+compile_error!("one feature must be enabled: ansi_term, nu-ansi-term, crossterm, gnu_legacy");
```

Comment:
@sylvestre @alexkunde Turns out this is problematic when I want to run `cargo publish`:
```
▶ cargo publish --dry-run 
    Updating crates.io index
   Packaging lscolors v0.15.0 (/home/shark/software/lscolors)
    Updating crates.io index
   Verifying lscolors v0.15.0 (/home/shark/software/lscolors)
   Compiling lscolors v0.15.0 (/home/shark/software/lscolors/target/package/lscolors-0.15.0)
error: one feature must be enabled: ansi_term, nu-ansi-term, crossterm, gnu_legacy
  --> src/bin.rs:14:1
   |
14 | compile_error!("one feature must be enabled: ansi_term, nu-ansi-term, crossterm, gnu_legacy");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: unused import: `Style`
 --> src/bin.rs:6:26
  |
6 | use lscolors::{LsColors, Style};
  |                          ^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: `lscolors` (bin "lscolors") generated 1 warning
error: could not compile `lscolors` (bin "lscolors") due to previous error; 1 warning emitted
error: failed to verify package tarball

```

Any ideas on how to resolve this?

Reply:
I think you just need to make one of those features enabled by default.  Right now even plain `cargo build` fails.

Reply:
sure, which one should be the default? nu-ansi-term?

Reply:
I submitted https://github.com/sharkdp/lscolors/pull/76 which makes that the default

Reply:
Thank you. @sylvestre New release is out. Let me know if things are not working as expected.

Reply:
thanks. it improved the coreutils. I am debugging some issues to know if it is this change in lscolors or in the coreutils 

thanks again